packages/account/src/Components/currency-selector/radio-button-group.tsx

```diff
@@ -3,17 +3,27 @@ import classNames from 'classnames';
 import { localize } from '@deriv/translations';
 import { Text } from '@deriv/components';
 
+export type TRadioButtonGroup = {
+    className: string;
+    is_fiat: boolean;
+    is_title_enabled?: boolean;
+    item_count: number;
+    label: string;
+    description: React.ReactNode;
+    has_fiat?: boolean;
+};
+
 const RadioButtonGroup = ({
     label,
     className,
     children,
-    is_title_enabled,
+    is_title_enabled = true,
     is_fiat,
     item_count,
     description,
     has_fiat,
-}) => {
-    const [is_currency_selected, setIsCurrencySelected] = useState(false);
+}: React.PropsWithChildren<TRadioButtonGroup>) => {
+    const [is_currency_selected, setIsCurrencySelected] = useState<boolean>(false);
```

Comment:
```suggestion
    const [is_currency_selected, setIsCurrencySelected] = useState(false);
```
TS will infer type from the value itself 🙂

Reply:
Comitted ur suggestion